### PR TITLE
Update mappings for assignments

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
This is actually Scott's PR! I accidentally merged his other [PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/706).
Dual Merge: 
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/300)

### CODE REVIEW

File: talentmap_api/fsbid/services/assignment_history.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def assignment_history_to_client_format(data):
         assignmentsCopy = data['results']
     if type(assignmentsCopy) is type([]):
         for x in assignmentsCopy:
-            pos = x.get('position', {})
+            pos = x.get('position', {})[0]
```
Comment (scott062) [inline talentmap_api/fsbid/services/assignment_history.py:46]: {} to [] for default

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/assignment_history.py:46]: 🧠 wow, so true - good catch!

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/assignment_history.py:46]: What about the `[0]`, you think that's ok?

File: talentmap_api/fsbid/services/assignment_history.py [hunk 2]
```diff
@@ -55,15 +55,15 @@ def assignment_history_to_client_format(data):
...
                             "position": {
-                                "grade": pos.get("pos_grade_code", None),
+                                "grade": pos.get("posgradecode", None),
                                 "skill": f"{pos.get('pos_skill_desc', None)} ({pos.get('pos_skill_code')})",
                                 "skill_code": pos.get("pos_skill_code", None),
                                 "bureau": f"({pos.get('pos_bureau_short_desc', None)}) {pos.get('pos_bureau_long_desc', None)}",
                                 "bureau_code": pydash.get(pos, 'bureau.bureau_short_desc'), # only comes through for available bidders
-                                "organization": pos.get('pos_org_short_desc', None),
-                                "position_number": pos.get('pos_num_text', None),
+                                "organization": pos.get('posorgshortdesc', None),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/assignment_history.py:63]: image
💯

File: talentmap_api/fsbid/services/assignment_history.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def assignment_history_to_client_format(data):
         assignmentsCopy = data['results']
     if type(assignmentsCopy) is type([]):
         for x in assignmentsCopy:
-            pos = x.get('position', {})
+            pos = pydash.get(x, 'position[0]') || {}
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/assignment_history.py:46]: That's embarrassing ...

**Pull request merged**